packages/block-library/src/buttons/edit.js

```diff
@@ -20,16 +23,42 @@ const alignmentHooksSetting = {
 	isEmbedButton: true,
 };
 
-function ButtonsEdit() {
+function ButtonsEdit( { attributes, setAttributes } ) {
+	const { blockDirection } = attributes;
```

Comment:
To be consistent with the navigation block, I think it'd be good to call this attribute `orientation`.

Reply:
Yeah, `orientation` seems clearer to me.

Reply:
Done. I've renamed the label as well. Thank you guys!❤️